docs/user-guide/global-configurations/image-promotion-policy.md

```diff
@@ -0,0 +1,136 @@
+
+4. A confirmation dialog box would appear. Click **Confirm**.
+
+### Performing Bulk Action
+
+You can also apply a policy to more than one application. Simply use the checkboxes to select the applications. You can do this even if there are many applications spanning multiple pages. You will see a draggable floating widget.
+
+Moreover, there are three filters available to make it easier for you to make the selections easier:
+* Application
+* Environment
+* Policy
+
+---
+
+## Result
+
+### Selecting Image for Promotion
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have build & deploy permission or above (along with access to the application and target environment) to select an image for promotion.
+{% endhint %}
+
+Here, you can promote images to the target environment(s).  
+
+1. Go to the **Build & Deploy** tab of your application.
+
+2. Click the **Promote** button next to the pipeline eligible for image promotion.
+
+3. In the `Select Image` tab, you will see a list of images. However, you can use the **Show Images from** dropdown to decide the image you wish to promote. It can be an image either from the CI pipeline or from any one of the intermediate stages (environments).
+
+4. Use the **SELECT** button on the image, and click **Promote to...**
+
+5. Select one or more target environments using the checkbox.
+
+6. Click **Promote Image**. 
+
+The image will get promoted to the target environment and it will be visible in the list of images in that target environment. However, if the super-admin enforced an approval mechanism, an email notification will be sent to the approvers. 
+
+You may check the status of your request in the `Approval Pending` tab.
+
+### Approving an Image Promotion Request
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+User needs to be a direct promotion approver or a super-admin to approve an image promotion request.
+{% endhint %} 
+
+1. Go to the **Build & Deploy** tab of your application.
+
+2. Click the **Promote** button next to the pipeline that has direct promotion enabled.
+
+3. Go to the `Approval Pending` tab to see the list of images requiring approval. By default, it shows a list of all images whose promotion request is pending with you.  
+
+{% hint style="info" %}
+All the images will show the source from which it is being promoted, i.e., CI stage or intermediate stage (environment).
+{% endhint %}
+
+4. Click **Approve for...** to choose the target environments to which it can be promoted.
+
+5. Click **Approve**.
+
+You can also use the **Show requests** dropdown to filter the image promotion requests for a specific target environment.
+
+### Deploying a Promoted Image
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have build & deploy permission or above (along with access to the application and target environment) to deploy an image.
```

Comment:
Users with build & deploy permission or above for the application and environment can deploy the promoted image.

Reply:
Footnote: If a user has approved the promotion request for an image, they may or may not be able to deploy depending upon the policy configuration. (Hyperlink to section)